src/olympia/discovery/data.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from django.utils.translation import string_concat, ugettext_lazy as _
+
+from olympia import amo
+
+
+class DiscoItem(object):
+    def __init__(self, *args, **kwargs):
+        self.addon_id = kwargs.get('addon_id')
+        self.type = kwargs.get('type')
+        self.heading = kwargs.get('heading')
+        self.description = kwargs.get('description')
```

Comment:
I started with a `namedtuple`, but then realized I'm going to need different properties for themes, plus some might be completely optional even for extensions, so I used a plain object instead.